node/consensus/data/data_clock_consensus_engine.go

```diff
@@ -551,8 +558,8 @@ func (e *DataClockConsensusEngine) Start() <-chan error {
 			frame = nextFrame
 
 			for i, trie := range e.GetFrameProverTries()[1:] {
-				if trie.Contains(e.provingKeyAddress) {
-					e.logger.Info("creating data shard ring proof", zap.Int("ring", i-1))
+				if trie.Contains(peerProvingKeyAddress) {
```

Comment:
Is this the last place or do we need to edit others ?

Reply:
last place